cypress/e2e/example.cy.js

```diff
@@ -2,50 +2,70 @@ import { faker } from '@faker-js/faker';
 
 describe('Bank app', () => {
-  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
-  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
-  const balance = depositAmount - withdrawAmount;
-  const user = 'Harry Potter';
-  const accountNumber = '1004';
+  const username = 'Hermoine Granger';
+  const account = '1001';
+  const accountBalance = 5096;
+  const depositAmount = faker.number.int(5000);
+  const balanceAfterDeposit = accountBalance + depositAmount;
+  const withdrawAmount = faker.number.int({ min: 1, max: balanceAfterDeposit });
+  const balanceAfterWithdraw = balanceAfterDeposit - withdrawAmount;
 
   before(() => {
-    cy.visit('/');
+    cy.visit('/#/login');
   });
 
   it('should provide the ability to work with bank account', () => {
     cy.contains('.btn', 'Customer Login').click();
-    cy.get('[name="userSelect"]').select(user);
+    cy.get('[name="userSelect"]').select(username);
     cy.contains('.btn', 'Login').click();
 
     cy.contains('[ng-hide="noAccount"]', 'Account Number')
-      .contains('strong', accountNumber)
+      .contains('strong', account)
       .should('be.visible');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', '0')
+    cy.contains('[ng-hide="noAccount"]', 'Balance : ')
+      .contains('strong', accountBalance)
       .should('be.visible');
-    cy.contains('.ng-binding', 'Dollar')
+    cy.contains('[ng-hide="noAccount"]', 'Currency')
+      .contains('strong', 'Dollar')
       .should('be.visible');
 
     cy.get('[ng-click="deposit()"]').click();
     cy.get('[placeholder="amount"]').type(depositAmount);
-    cy.contains('[type="submit"]', 'Deposit').click();
-
+    cy.contains('button[type="submit"]', 'Deposit').click();
     cy.get('[ng-show="message"]')
       .should('contain', 'Deposit Successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', depositAmount)
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance : ')
+      .contains('strong', balanceAfterDeposit)
       .should('be.visible');
 
     cy.get('[ng-click="withdrawl()"]').click();
-    cy.contains('[type="submit"]', 'Withdraw')
-      .should('be.visible');
-    cy.get('[placeholder="amount"]').type(withdrawAmount);
-    cy.contains('[type="submit"]', 'Withdraw').click();
 
+    cy.get('[ng-model="amount"]').type(withdrawAmount);
+    cy.contains('button[type="submit"]', 'Withdraw').click();
     cy.get('[ng-show="message"]')
       .should('contain', 'Transaction successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', balance)
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance : ')
+      .contains('strong', balanceAfterWithdraw)
       .should('be.visible');
+
+    cy.get('[ng-click="transactions()"]').click();
+    cy.get('[ng-click="back()"]').click();
+
+    cy.get('[ng-click="transactions()"]').click();
+    cy.get('#start').type('2024-10-14T18:20:00');
```

Comment:
The date format used in `type('2024-10-14T18:20:00')` might not be compatible with the input field. Ensure the input field accepts this format or adjust accordingly.